root/report/recordings_same_name_different_artists_same_name.tt

```diff
@@ -0,0 +1,19 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings with the same name by different artists with the same name') full_width=1 -%]
+
+<h1>[% l('Recordings with the same name by different artists with the same name') %]</h1>
+
+<ul>
+    <li>[% l('This report shows all recordings with the same name that have different artist MBIDs with the same name.') %]</li>
```

Comment:
Grammar nitpick: "different artist MBIDs with the same name" means the names of the MBIDs themselves, which doesn't make much sense. I would put the explanation in parentheses like "different artists (having different MBIDs) with the same name."